results/final_results.json

```diff
@@ -15,9 +15,9 @@
         "standard": 16.43666922426724
     },
     "htmcore": {
-        "reward_low_FN_rate": 66.1922106936328,
-        "reward_low_FP_rate": 58.7930712907694,
-        "standard": 63.081419488725054
+        "reward_low_FN_rate": 76.5626293570994,
+        "reward_low_FP_rate": 61.359926511549155,
+        "standard": 71.3094612770284
```

Comment:
this is with the "master" of htmcore? Are the results stable, ie reasonably consistent over several runs? If so, please also bump our score table in Readme and you can move us to the 1st place! ;)

Reply:
wait a second, i must push the non spatial numbers &try to build with latest htm.core